geonode/templates/metadata_detail.html

```diff
@@ -1,393 +1,393 @@
-{% extends "geonode_base.html" %}

```

Comment:
You completely changed the format of this file, it is impossible to understand what's changed!